_posts/en/pages/2016-01-01-about.md

```diff
@@ -10,11 +10,15 @@ version: 1
 redirect_from:
   - /zh_TW/about/
 ---
-
 Bitcoin Core is an [open source](https://opensource.org/) project which maintains and releases Bitcoin client software called "Bitcoin Core".
 
 It is a direct descendant of the original Bitcoin software client released by Satoshi Nakamoto after he published the famous [Bitcoin whitepaper](/bitcoin.pdf).
 
-Bitcoin Core consists of both "full-node" software for fully validating the blockchain as well as a bitcoin wallet. The project also currently maintains related software such as the cryptography library [libsecp256k1](https://github.com/bitcoin/secp256k1) and others located at [GitHub](https://github.com/bitcoin).
+## Our Mission
+Bitcoin Core supports the creation of a sustainable Bitcoin ecosystem by providing code and resources to the Bitcoin environment. Working together, Bitcoin Core, and its users, form the most ambitious and successful investment in the deployment of blockchain technology.
 
 Anyone can [contribute to Bitcoin Core](/en/contribute/).
+
+## Getting Started
+Bitcoin Core consists of both "full-node" software for fully validating the blockchain as well as a bitcoin wallet. The project also currently maintains related software such as the cryptography library [libsecp256k1](https://github.com/bitcoin/secp256k1) and others located at [GitHub](https://github.com/bitcoin).
```

Comment:
I wouldn't call this section "Getting started"

Reply:
Sure. I can change that. any suggestions? Here's a few: "Our Software", "To Learn More", "Additional Projects", "Additional Repositories",... (need more?)